app/controllers/support/devices/order_status_controller.rb

```diff
@@ -15,6 +15,12 @@ def update
         allocation.cap = allocation.cap_implied_by_order_state(order_state: @form.order_state, given_cap: @form.cap)
         allocation.save!
         @school.update!(order_state: @form.order_state)
+
+        request = Computacenter::OutgoingAPI::CapUpdateRequest.new(allocation_ids: [allocation.id])
+        request.post!
```

Comment:
As mentioned on the other PR, should we extract this logic into a service or model?

Reply:
Yes, totally

Reply:
@benilovj done!